jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -727,6 +795,39 @@
 					"response": []
 				},
 				{
+					"name": "create webserver2 for duplicate error check",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 201\"] = responseCode.code === 201;",
```

Comment:
Same thing: the test is "duplicate error check" but it looks like it's expecting a successful creation of a web server